core/src/main/scala/com.snowplowanalytics.stream/loader/model.scala

```diff
@@ -74,10 +74,11 @@ package model {
     username: Option[String],
     password: Option[String],
     maxTimeout: Long,
+    maxRetries: Int,
```

Comment:
this should have also been in its own commit

Reply:
removed